analytics_data_api/v0/views/learners.py

```diff
@@ -168,7 +181,7 @@ def _validate_query_params(self):
 
     def get_queryset(self):
         """
-        Fetches the user list from elasticsearch.  Note that an
+        Fetches the user list from elasticsearch. Note that an
```

Comment:
that said, I can re-edit this in a different editor if you feel strongly about it, @dan-f !